2024-03-05 06:36:39,234 INFO    MainThread:25624 [wandb_setup.py:_flush():76] Current SDK version is 0.16.3
2024-03-05 06:36:39,237 INFO    MainThread:25624 [wandb_setup.py:_flush():76] Configure stats pid to 25624
2024-03-05 06:36:39,241 INFO    MainThread:25624 [wandb_setup.py:_flush():76] Loading settings from C:\Users\16979\.config\wandb\settings
2024-03-05 06:36:39,241 INFO    MainThread:25624 [wandb_setup.py:_flush():76] Loading settings from G:\My Drive\SI_630\Homeworks\Homework2 - Backup\wandb\settings
2024-03-05 06:36:39,241 INFO    MainThread:25624 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2024-03-05 06:36:39,241 INFO    MainThread:25624 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2024-03-05 06:36:39,242 INFO    MainThread:25624 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2024-03-05 06:36:39,244 INFO    MainThread:25624 [wandb_init.py:_log_setup():526] Logging user logs to G:\My Drive\SI_630\Homeworks\Homework2 - Backup\wandb\run-20240305_063639-b3gh2lhi\logs\debug.log
2024-03-05 06:36:39,246 INFO    MainThread:25624 [wandb_init.py:_log_setup():527] Logging internal logs to G:\My Drive\SI_630\Homeworks\Homework2 - Backup\wandb\run-20240305_063639-b3gh2lhi\logs\debug-internal.log
2024-03-05 06:36:39,246 INFO    MainThread:25624 [wandb_init.py:_jupyter_setup():472] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x00000185AB707AA0>
2024-03-05 06:36:39,246 INFO    MainThread:25624 [wandb_init.py:init():566] calling init triggers
2024-03-05 06:36:39,246 INFO    MainThread:25624 [wandb_init.py:init():573] wandb.init called with sweep_config: {}
config: {}
2024-03-05 06:36:39,247 INFO    MainThread:25624 [wandb_init.py:init():616] starting backend
2024-03-05 06:36:39,247 INFO    MainThread:25624 [wandb_init.py:init():620] setting up manager
2024-03-05 06:36:39,278 INFO    MainThread:25624 [backend.py:_multiprocessing_setup():105] multiprocessing start_methods=spawn, using: spawn
2024-03-05 06:36:39,279 INFO    MainThread:25624 [wandb_init.py:init():628] backend started and connected
2024-03-05 06:36:39,357 INFO    MainThread:25624 [wandb_run.py:_label_probe_notebook():1295] probe notebook
2024-03-05 06:36:39,362 INFO    MainThread:25624 [wandb_run.py:_label_probe_notebook():1305] Unable to probe notebook: 'charmap' codec can't decode byte 0x81 in position 3711: character maps to <undefined>
2024-03-05 06:36:39,362 INFO    MainThread:25624 [wandb_init.py:init():720] updated telemetry
2024-03-05 06:36:39,470 INFO    MainThread:25624 [wandb_init.py:init():753] communicating run to backend with 90.0 second timeout
2024-03-05 06:36:40,158 INFO    MainThread:25624 [wandb_run.py:_on_init():2262] communicating current version
2024-03-05 06:36:40,547 INFO    MainThread:25624 [wandb_run.py:_on_init():2271] got version response 
2024-03-05 06:36:40,547 INFO    MainThread:25624 [wandb_init.py:init():804] starting run threads in backend
2024-03-05 06:36:40,866 INFO    MainThread:25624 [wandb_run.py:_console_start():2241] atexit reg
2024-03-05 06:36:40,867 INFO    MainThread:25624 [wandb_run.py:_redirect():2096] redirect: wrap_raw
2024-03-05 06:36:40,870 INFO    MainThread:25624 [wandb_run.py:_redirect():2161] Wrapping output streams.
2024-03-05 06:36:40,870 INFO    MainThread:25624 [wandb_run.py:_redirect():2186] Redirects installed.
2024-03-05 06:36:40,875 INFO    MainThread:25624 [wandb_init.py:init():847] run started, returning control to user process
2024-03-05 07:18:12,848 INFO    MainThread:25624 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-05 07:18:12,848 INFO    MainThread:25624 [wandb_init.py:_pause_backend():437] pausing backend
2024-03-05 07:18:16,057 INFO    MainThread:25624 [wandb_init.py:_resume_backend():442] resuming backend
2024-03-05 07:18:16,170 INFO    MainThread:25624 [jupyter.py:save_ipynb():373] not saving jupyter notebook
2024-03-05 07:18:16,171 INFO    MainThread:25624 [wandb_init.py:_pause_backend():437] pausing backend
2024-03-05 07:18:23,780 WARNING MsgRouterThr:25624 [router.py:message_loop():77] message_loop has been closed
